docs/edgeql/overview.rst

```diff
@@ -86,6 +86,29 @@ behavior.
 A *simple path* is a path which begins with a set reference.
 
 
+.. _ref_eql_fundamentals_name_resolution:
+
+Name Resolution
+===============
+
+In EdgeQL a name can either be *fully-qualified*, i.e. of the form
+``module_name::entity_name`` or in short form of just ``entity_name``
+(for more details see :ref:`ref_eql_lexical_names`). Any short name is
+ultimately resolved to some fully-qualified name in the following
+manner:
+
+1) Look for a match to the short name in the current module (typically
+   ``default``, but it can be changed).
+2) Look for a match to the short name in the ``std`` module.
+
+Normally the current module is called ``default``, which is
+automatically created in any new database. It is possible to override
+the current module globally on the session level with a ``SET
+MODULE blah`` :ref:`command <ref_eql_statements_session_set_alias>`. It is
```

Comment:
We should decide on a dictionary of placeholder names but my intuition is that they should **not** be `blah`, `foo`, `bar`, and so on.

Reply:
I'll swap it to "my_module" in this example, I think.

Reply:
@ambv good point.